third_party/move/move-compiler-v2/src/file_format_generator/peephole_optimizer/inefficient_loads.rs

```diff
@@ -61,8 +64,12 @@ impl WindowOptimizer for InefficientLoads {
                     // We have reached the end of the pattern (point 4 in the module documentation).
                     let sequence = &window[2..index + 2];
                     let load_constant = &window[0..1];
+                    let transformed_code = [sequence, load_constant].concat();
+                    let original_offsets = (2..(index + 2) as CodeOffset)
+                        .chain(iter::once(0))
+                        .collect::<Vec<_>>();
```

Comment:
Could we write this more intelligibly as 
```
let original_offsets = [[std::ops::Range<CodeOffset> { start: 2, end: index+2}, [0]].concat();
```
or perhaps just add a comment:
```
// original_offsets = [2, ..., index+2, 0];
```

Reply:
done, added a comment